src/App.js

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(({ id }) => id === todo.userId),
+}));
 
 function App() {
   return (
     <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
+      <TodoList todos={preparedTodos} users={users} />
```

Comment:
no need to pass users, just import it in file where you need it

Reply:
prepare todos before passing it to TodoList, each todo should have a user property with object 
also put todos in App's state and remove them from TodoList state